styles.css

```diff
@@ -503,3 +515,129 @@ p {
     margin-top: 20px;
   }
 }
+
+.contact-form {
+  margin-top: 30px;
+  width: 375px;
+  height: 779px;
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  background-attachment: scroll;
+  background-color: #6070ff;
+  width: 100%;
+  border-top-left-radius: 8rem;
+  font-size: 40px;
+  font-weight: 700;
+}
+
+.contact-bg-img {
+  background-image: url('./Images/contact\ form\ shapes-mobile@2x.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  height: auto;
+}
+
+.home-indicator {
+  background-image: url('./Images/Home\ Indicator/Shape.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 25%;
+  margin-top: 1.5rem;
+  margin-left: 8rem;
+  margin-bottom: 1rem;
+  height: 0.4rem;
+}
+
+.contact-p {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  height: 137px;
+  font-size: 1.2rem;
+  font-weight: 400;
+  font-style: normal;
+  color: #ebebff;
+  margin-top: 5rem;
+  margin-bottom: 1.5rem;
+  margin-left: 18px;
+  text-align: center;
+  line-height: 1.5rem;
+}
+
+input,
+textarea {
+  display: flex;
+  flex-direction: column;
+  height: 3rem;
+  padding-left: 0.5rem;
+  align-items: center;
+  width: 80%;
+  margin: 12px 24px 11px 28px;
+  background-color: #fff;
+  margin-left: 2rem;
+  margin-right: 1rem;
+  margin-bottom: 0.8rem;
+  border-width: 1px;
+  border: 0.1rem solid #acb7c3;
+  border-radius: 0.5rem;
+  border-color: #cfd8dc;
+  font-size: 0.9rem;
+  font-weight: 500;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+}
+
+textarea {
+  font-family: "Poppins", sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  text-align: left;
+  height: 10rem;
+  border-radius: 8px;
+  margin: 12px 24px 11px 28px;
+  background-color: #fff;
+  padding-top: 1rem;
+  border-width: 1px;
+  border-color: #cfd8dc;
+}
+
+.contact button {
+  border: 0.01rem;
+  height: 2.5rem;
+  width: 28%;
+  border-radius: 0.5rem;
+  margin-left: 2rem;
+  margin-top: 1rem;
+  color: #6070ff;
+  background-color: #fff;
+  box-shadow: #fff;
+  font-size: 1rem;
+  font-weight: 500;
+  font-style: normal;
+  text-align: center;
+}
+
+.contact button :hover {
+  background-color: blue;
+  color: white;
+}
+
+.contact button :active {
+  color: #6070ff;
+}
+
+.contact-mobile-btn:disabled {
+  color: #fff;
+}
```

Comment:
- [ ] Good job adding the hover, active and disabled effects. Kindly add the `pressed` effect as well to meet the requirements.
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/81932478/0572a23b-74ca-4eff-9e9a-b62769ee9789)